packages/TelegramClient-UI.package/TCUMessage.class/class/newFrom..st

```diff
@@ -0,0 +1,11 @@
+instance creation
+newFrom: aCoreMessage
+
+	^ (aCoreMessage class caseOf: {
+			[TCCTextMessage] -> [TCUTextMessage].
+			[TCCPhotoMessage] -> [TCUPhotoMessage].
+		} otherwise: [TCUTextMessage])
```

Comment:
I would prefer the `}` to be a the same indent level as the `^`, but this is also acceptable